frontend/src/components/modal/share/components/ShareModalContent.jsx

```diff
@@ -7,15 +7,15 @@ import ShareModalLink from './ShareModalLink';
 import Links from '../Links';
 
 const ShareModalContent = () => (
-	<ShareModalContainer onClick={(e) => e.stopPropagation()}>
-		<CloseButton />
-		<ShareModalHeader />
-		<ShareModalLinkContainer>
-			{Links.map((link, i) => (
-				<ShareModalLink key={i} link={link} />
-			))}
-		</ShareModalLinkContainer>
-	</ShareModalContainer>
+    <ShareModalContainer onClick={(e) => e.stopPropagation()}>
+        <CloseButton />
+        <ShareModalHeader />
+        <ShareModalLinkContainer>
+            {Links.map((link, i) => (
+                <ShareModalLink key={i} link={link} />
+            ))}
+        </ShareModalLinkContainer>
+    </ShareModalContainer>
 );
 
```

Comment:
If you aren't using TS or Flow, I suggest using propTypes to add type safety to your react components. It also helps to make your component more readable/clear what should be passed in

Reply:
Good point. I can start incorporating it for future projects.